libs/withdraws/src/setup-tests.ts

```diff
@@ -1,4 +1,5 @@
 import '@testing-library/jest-dom';
+import '../../../setup-test';
```

Comment:
I think we should probably set these set up in the jest.config.ts, rather than importing here

Reply:
I couldn't find any other easy way of share jest setup configuration :-(
Moving it to jest.config caused crash because we cannot import from over local rootDir